### PR TITLE
Added sample of NodeJS application that requires DB

### PR DESCRIPTION
**Pull request by sleshchenko**
This PR adds a sample of NodeJS application that requires DB

### CODE REVIEW

File: samples/web-nodejs-with-db-sample/devfile.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+specVersion: 0.0.1
+name: multiple-k8s-tools
```
Comment (mshaposhnik) [inline samples/web-nodejs-with-db-sample/devfile.yaml]: Maybe name it smthng like `nodejs-with-db` ?

**Comment by mshaposhnik** [comment]
It seems we should use `devfile.yml` not `.yaml` (https://github.com/eclipse/che/issues/12837)

**Review by mshaposhnik — APPROVED** [review]
other is LGTM

File: samples/web-nodejs-with-db-sample/devfile.yaml (hunk 1 above)
Comment (sleshchenko) [reply samples/web-nodejs-with-db-sample/devfile.yaml]: Good catch.

**Comment by sleshchenko** [comment]
> It seems we should use devfile.yml not .yaml (eclipse/che#12837)

You're right. Fixed.
YAML file extension is also fixed in `theia-hello-world-frontend-plugin` sample

**Comment by l0rd** [comment]
Nice work @sleshchenko 

Are the services and routes of the 2 YAMLs used in the Che workspace? I mean are we creating a workspace server corresponding to the nodejs route?

I am asking because if we do not use these kubernetes resources in a Che workspace we may want to filter them out (and make both tools of type kubernetes). And anyhow routes should be automatically setup once https://github.com/eclipse/che/issues/12061 is completed.

**Comment by sleshchenko** [comment]
@l0rd 

> Are the services and routes of the 2 YAMLs used in the Che workspace? I mean are we creating a workspace server corresponding to the nodejs route?

All objects are created. Mongo DB service is used, as you can see here, application access to database via `mongo` host 
https://github.com/sleshchenko/devfile/blob/nodejs-app-with-db/samples/web-nodejs-with-db-sample/devfile.yml#L26

As I showed in my Demo https://youtu.be/e4UrtbTmV7M, route is not accessible for a user from Theia, but he is able to get it from OpenShift Console.

I like the idea to make sample usable for Kubernetes infra too, but until https://github.com/eclipse/che/issues/12061 is not complited, removing route is not a good thing to do because user won't be able to access NodeJS Application.
Also, using OpenShift tools shows an ability to use OpenShift template with default parameters, maybe it should be done as a dedicated sample.

WDYT? Do you think it should be removed now or after resolving of https://github.com/eclipse/che/issues/12061?

**Review by skabashnyuk — APPROVED** [review]

**Comment by l0rd** [comment]
Thank you @sleshchenko for the explanation I think we can merge it like that

**Review by l0rd — APPROVED** [review]

**Pull request merged**